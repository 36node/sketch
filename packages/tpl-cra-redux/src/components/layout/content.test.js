import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Content from "./content";

it("should renders correctly", () => {
  const tree = renderer.create(<Content> somthing </Content>).toJSON();
  expect(tree).toMatchSnapshot();
});
