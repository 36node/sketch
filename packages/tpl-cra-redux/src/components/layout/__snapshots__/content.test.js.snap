// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
.c0 {
  padding: 0 12px;
}

<div
  className="c0 ant-layout-content"
>
   somthing 
</div>
`;
